src/content/reference/react/createContext.md

```diff
@@ -168,7 +168,7 @@ export const ThemeContext = createContext('light');
 export const AuthContext = createContext(null);
 ````
 
-Components declared in other files can then use the [`import`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import) statement to read or provide this context:
+Komponen yang dideklarasikan di berkas lain dapat menggunakan pernyataan [`import`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import) untuk membaca atau menyediakan konteks ini:
```

Comment:
Perhaps the use of the word `berkas` here needs to be reconfirmed, as some remain in `file` or `files`.

Reply:
Back to file ->`*file*`